tests/test_Spectrum.py

```diff
@@ -382,6 +398,35 @@ def testReadWriteFits(self):
         else:
             os.unlink(filename)
 
+    def testPlot(self):
+        """Test plotting of spectra
+
+        Not easy to test the actual result, but we can test that the API hasn't
+        been broken.
+        """
+        import matplotlib.pyplot as plt
+        plt.switch_backend("agg")  # In case someone has loaded a different backend that will cause trouble
+        num = 5
+        spectra = self.makeSpectrumSet(num)
+        # Write directly to file
+        with lsst.utils.tests.getTempFilePath(".png") as filename:
+            spectra.plot(numRows=4, filename=filename)
+        # Check return values
+        with lsst.utils.tests.getTempFilePath(".png") as filename:
+            numRows = 4  # Must be > 1 for len(axes) to work
+            figure, axes = spectra.plot(numRows=numRows)
+            figure.savefig(filename)
+            self.assertEqual(len(axes), numRows)
+        # Test one row, write directly to file
+        with lsst.utils.tests.getTempFilePath(".png") as filename:
+            spectra.plot(numRows=1, filename=filename)
+        # Test one row, check return values
+        with lsst.utils.tests.getTempFilePath(".png") as filename:
+            figure, axes = spectra.plot(numRows=1)
+            with self.assertRaises(TypeError):
+                axes[0]
+            figure.savefig(filename)
+
 
 class TestMemory(lsst.utils.tests.MemoryTestCase):
```

Comment:
(Outside the context of this PR I know) but why is this class empty?

Reply:
It's so that pytest can find it. It's part of the LSST test suite boilerplate.

Reply:
Ok thanks